src/.nuget/Microsoft.NETCore.ILAsm/runtime.Linux.Microsoft.NETCore.ILAsm.props

```diff
@@ -1,11 +1,9 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <SkipPackageFileCheck>true</SkipPackageFileCheck>
-    <PackageTargetRuntime>linux-$(PackagePlatform)</PackageTargetRuntime>
-    <!-- only build for x64 -->
     <PackagePlatforms>x64;</PackagePlatforms>
+    <PackagePlatforms Condition="'$(RuntimeOS)' == 'ubuntu.14.04'">x64;arm;</PackagePlatforms>
```

Comment:
I think we can remove PackagePlatforms.  This was an older version of filtering where we dummy-ed out the BuildTarget when PackagePlatforms didn't contain the build Platform.  We no longer use this in CoreFx, we should remove it here as well.  We're doing more advanced filtering in the builds file.